Create 2020-11-12-brian-hills.md (#29)
USRSE/rse-stories: _posts/2020-11-12-brian-hills.md

---
layout: post
title: "Key is your value set"
author: "@pweschmidt"
rse: "Brian Hills"
phenotype: 
excerpt: "Meet Brian Hills, Deputy CEO and Head of Service Design at the Data Lab in Scotland. After years in the private sector Brian joined Datalab to help bridge the gap in data science between universities and the private sector. Ethics in data plays an important part and success strongly depends on your value set, says Brian."
date: 2020-11-12 8:30:00
external_media: https://us-rse.org/rse-stories-episodes-1/2020/rse-stories-brian-hills-episode-42.mp3
length: 13651300
duration: "00:28:27"
explicit: "no"
resources:
  - name: The Data Lab
    url: https://www.thedatalab.com
  - name: Brian's Twitter account
    url: https://twitter.com/brianhills 
--- 

> In this episode I talk to [Brian Hills](https://twitter.com/brianhills) deputy CEO and Head of Service Design for the [DataLab](https://www.thedatalab.com) in Scotland. Brian joined the team after many years in the private sector. And his experiences there helps him work with universities and companies to bridge the gap in skills we have today. Working in AI/ML is often misunderstood. Many downplay the importance of good, solid data engineering. As Brian points out, he had companies remarking that new graduates often want to work on algorithms rather than "cleaning up the data" for processing. 

Reviewer: ```suggestion
In this episode we talk to [Brian Hills](https://twitter.com/brianhills) deputy CEO and Head of Service Design for the [DataLab](https://www.thedatalab.com) in Scotland. Brian joined the team after many years in the private sector. And his experiences there help him work with universities and companies to bridge the gap in skills we have today. Working in AI/ML is often misunderstood. Many downplay the importance of good, solid data engineering. As Brian points out, he had companies remarking that new graduates often want to work on algorithms rather than "cleaning up the data" for processing. 
```

> Ethics in data science also plays a key role. With data science playing an increasingly important role in all our lives engineers working in the field need to employ a solid set of ethical values and use proper toolkits.

Reviewer: ```suggestion
Ethics in data science also play a key role. With data science playing an increasingly important role in all our lives engineers working in the field need to employ a solid set of ethical values and use proper toolkits.
```

---
layout: post
title: "Key is your value set"
author: "@pweschmidt"
rse: "Brian Hills"
phenotype: 
excerpt: "Meet Brian Hills, Deputy CEO and Head of Service Design at the Data Lab in Scotland. After years in the private sector Brian joined Datalab to help bridge the gap in data science between universities and the private sector. Ethics in data plays an important part and success strongly depends on your value set, says Brian."
date: 2020-11-12 8:30:00
external_media: https://us-rse.org/rse-stories-episodes-1/2020/rse-stories-brian-hills-episode-42.mp3
length: 13651300
duration: "00:28:27"
explicit: "no"
resources:
  - name: The Data Lab
    url: https://www.thedatalab.com
  - name: Brian's Twitter account
    url: https://twitter.com/brianhills 

> 

Reviewer: ```suggestion
```
